spec.bs

```diff
@@ -569,3 +569,50 @@ Alternatively, we can add the snippet at the beginning of the following HTML spe
 </div>
 
 <p class="note">[=Validate a module script URL=] is applied to all module URLs before they start loading, even in paths where [=resolve a module specifier=] and import maps are not applied (e.g. `<script src="..." type="module">`).</p>
+
+<hr>
+<h2 id="internet-media-type-registration">Appendix A: Internet Media Type Registration</h2>
+
+This appendix provides the provisional registration of the Internet Media Type `application/importmap+json` in accordance with [[RFC6838]].
+
+: Type name:
+:: application
+: Subtype name:
+:: importmap+json
+: Required parameters:
+:: N/A
+: Optional parameters:
+:: N/A
+: Encoding considerations:
+:: 8bit
+: Security considerations:
+:: Since Import Maps have the ability to direct which module resolutions are to be provided to all ECMAScript module imports of a given ECMAScript environment, control of the Import Map should be considered to be execution-level application access. Integration with existing policy systems, including for example CORS and CSP, can be used to mitigate and restrict unwanted target URLs from being executed. In addition, the specification states that only those Import Maps served to browsers with the `application/importmap+json` MIME type will be executed.
```

Comment:
```suggestion
:: Since import maps have the ability to direct which module resolutions are to be provided to all JavaScript module imports of a given JavaScript environment, control of the import map should be considered to be execution-level application access. Integration with existing policy systems, including for example CORS and CSP, can be used to mitigate and restrict unwanted target URLs from being executed. In addition, the specification states that only those import maps served to browsers with the `application/importmap+json` MIME type will be executed.
```